heron/stmgr/src/cpp/manager/stmgr-clientmgr.cpp

```diff
@@ -114,6 +117,8 @@ StMgrClient* StMgrClientMgr::CreateClient(const sp_string& _other_stmgr_id,
   options.set_max_packet_size(config::HeronInternalsConfigReader::Instance()
                                   ->GetHeronStreammgrNetworkOptionsMaximumPacketMb() *
                               1024 * 1024);
+  options.set_high_watermark(high_watermark_);
```

Comment:
Should we pass high_watermark and low_watermark from constructor, or similar to max_packet_size, reading config inside CreateClient(...) method?

On the one hand, it is not necessary to change the signature of a class's constructor. On the other hand, It will match the style of max_packet_size.